aesara/tensor/subtensor.py

```diff
@@ -1235,33 +1235,52 @@ def set_subtensor(x, y, inplace=False, tolerate_inplace_aliasing=False):
 
 
 def inc_subtensor(
-    x, y, inplace=False, set_instead_of_inc=False, tolerate_inplace_aliasing=False
+    x,
+    y,
+    inplace=False,
+    set_instead_of_inc=False,
+    tolerate_inplace_aliasing=False,
+    ignore_duplicates=False,
 ):
-    """
-    Return x with the given subtensor incremented by y.
+    """Update the value of an indexed array by a given amount.
+
+    This is equivalent to ``x[indices] += y`` or ``np.add.at(x, indices, y)``,
+    depending on the value of `ignore_duplicates`.
 
     Parameters
     ----------
     x
         The symbolic result of a Subtensor operation.
     y
-        The amount by which to increment the subtensor in question.
+        The amount by which to increment the array.
     inplace
-        Don't use. Aesara will do it when possible.
+        Don't use.  Aesara will do in-place operations itself, when possible.
     set_instead_of_inc
         If True, do a set_subtensor instead.
     tolerate_inplace_aliasing:
-        Allow x and y to be views of a single underlying array even while
-        working inplace. For correct results, x and y must not be overlapping
-        views; if they overlap, the result of this Op will generally be
-        incorrect. This value has no effect if inplace=False.
+        Allow `x` and `y` to be views of a single underlying array even while
+        working in-place. For correct results, `x` and `y` must not be overlapping
+        views; if they overlap, the result of this `Op` will generally be
+        incorrect. This value has no effect if ``inplace=False``.
+    ignore_duplicates
+        This determines whether or not ``x[indices] += y`` is used or
+        ``np.add.at(x, indices, y)``.
```

Comment:
We might want to have a small note over here stating that it's the `x[indices] += y` operation that ignores the duplicates and is faster than the `add.at`. Otherwise LGTM!